### ISSUE TITLE
Override system variables

### ISSUE THREAD

**Issue by doasync** [entropitor/dotenv-cli#46]
> **What happens to environment variables that were already set?**
> We will never modify any environment variables that have already been set. In particular, if there is a variable in your .env file which collides with one that already exists in your environment, then that variable will be skipped. This behavior allows you to override all .env configurations with a machine-specific environment, although it is not recommended.
> 

Add an **option** to ignore environment vars and set the variables from a file.

For example a user has `NODE_ENV=development` set in his system. Jest will fail on that. I want to be able to ignore system vars and set explicitly all variables that I need for my script/command (`NODE_ENV=test`)

**Comment by entropitor** [comment]
@doasync Is there an option to do this using `dotenv` (from where you linked the documentation)? 

If not, I think it's out of scope for this package. 

I'm gonna close it for now, but feel free to link to documentation / open a PR and I'll re-open this ticket.

**Comment by doasync** [comment]
There is a feature request in `dotenv`:
https://github.com/motdotla/dotenv/issues/436

**Comment by tjohnsonBCM** [comment]
If anyone is looking for an override option, I've added it in a new package [dotenv-override-cli](https://www.npmjs.com/package/dotenv-override-cli).

@entropitor I'd be happy to contribute a PR to this repo adding the feature if you'd like. To implement an override option I didn't need any changes to `dotenv`, but did need to make changes to `dotenv-expand`. I resorted to bundling a patched version of `dotenv-expand` in my package because they don't seem to be actively accepting PRs.
